src/app/store.ts

```diff
@@ -1,10 +1,10 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
+import { todosSlice } from '../features/todos';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(currentTodoSlice, filterSlice, todosSlice);
```

Comment:
Replace `combineSlices` with `combineReducers` to correctly combine the reducers from different slices.